rest-modules/user/index.js

```diff
@@ -1,7 +1,41 @@
+/**
+ * Implements routes:
+ * /users - get list of users
+ * /user/id/:id - get user by ID
```

Comment:
I think we do not need extra url part, it would be better to have endpoint like /user/:id